projects/schematics/src/migrations/5_0/constructor-deprecations/data/cart-lib-checkout.migration.ts

```diff
@@ -0,0 +1,2679 @@
+      },
+      {
+        className: 'CartConfigService',
+        importPath: '@spartacus/cart/base/core',
+      },
+    ],
+  },
+  {
+    class: 'CartItemListComponent',
+    importPath: '@spartacus/storefront',
+    deprecatedParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'SelectiveCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'UserIdService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'MultiCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'SelectiveCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'MultiCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'ActiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+      {
+        className: 'SelectiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+      {
+        className: 'MultiCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+      {
+        className: 'ChangeDetectorRef',
+        importPath: '@angular/core',
+      },
+      {
+        className: 'OutletContextData',
+        importPath: '@spartacus/storefront',
+      },
+    ],
+  },
+  {
+    class: 'CartPageLayoutHandler',
+    importPath: '@spartacus/storefront',
+    deprecatedParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'SelectiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'SelectiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'ActiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+      {
+        className: 'SelectiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+      {
+        className: 'CartConfigService',
+        importPath: '@spartacus/cart/base/core',
+      },
+    ],
+  },
+  {
+    class: 'CartTotalsComponent',
+    importPath: '@spartacus/storefront',
+    deprecatedParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'ActiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'CartTotalsComponent',
```

Comment:
the previous one is also `CartTotalsComponent`

Reply:
https://github.com/SAP/spartacus/pull/15305/files/ee8c9d62a1c0c6ef60875a30011f915b2c79446b#r821010650